diff --git a/configure b/configure
index 7bee4db..bdeb62f 100755
--- a/configure
+++ b/configure
@@ -14109,6 +14109,7 @@ $as_echo "$as_me: WARNING:
 fi
 
 fi
+
 if test "$use_descriptor_passing" = "yes"; then
 
 $as_echo "#define USE_DESCRIPTOR_PASSING 1" >>confdefs.h
@@ -14293,9 +14294,9 @@ $as_echo "$as_me: Use gpgrt-config with $gpgrt_libdir as gpg-error-config" >&6;}
     req_minor=`echo $min_gpg_error_version | \
                sed 's/\([0-9]*\)\.\([0-9]*\)/\2/'`
     major=`echo $gpg_error_config_version | \
-               sed 's/\([0-9]*\)\.\([0-9]*\).*/\1/'`
+               sed 's/\([0-9]*\)\.\([0-9]*\).*/\1/' | cut -f 3 -d' ' `
     minor=`echo $gpg_error_config_version | \
-               sed 's/\([0-9]*\)\.\([0-9]*\).*/\2/'`
+               sed 's/\([0-9]*\)\.\([0-9]*\).*/\2/' | cut -f 3 -d' '`
     if test "$major" -gt "$req_major"; then
         ok=yes
     else
@@ -14420,7 +14421,6 @@ $as_echo "no" >&6; }
 
 
 
-
 #
 # Checks for library functions.
 #
